blazor/file-manager/upload.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: Upload in Blazor FileManager Component | Syncfusion
+description: Checkout and learn here all about available Upload in Syncfusion Blazor FileManager component and much more.
+platform: Blazor
+control: File Manager
+documentation: ug
+---
+
+# Upload in Blazor FileManager component
+
+The Blazor FileManager component provides a [FileManagerUploadSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerUploadSettings.html) feature that allows users to upload files to the FileManager. It provides various settings and options to customize how files are uploaded, such as controlling file size, restricting file types, and enabling chunk uploads.
```

Comment:
The Blazor FileManager component provides a [FileManagerUploadSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.FileManager.FileManagerUploadSettings.html) property with various options to customize how files are uploaded, such as controlling file size, restricting file types, and enabling chunk uploads.

Reply:
modified the line and updated.